lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -850,6 +861,13 @@ public static function get_resolved_theme_uris( $theme_json ) {
 			return $resolved_theme_uris;
 		}
 
+		$theme_json_data               = $theme_json->get_raw_data();
+		$resolved_theme_uris_cache_key = md5( wp_json_encode( $theme_json_data ) );
```

Comment:
Good point, but I'm not sure whether it's required here just yet. 

The method is called several times in a single session and the main performance enhancement here is to avoid subsequent (and expensive) processing in the same session. This applies to the frontend and editor.

Nothing is saved to the database so I don't expect theme developers to run into any gotchas when testing sites given that the cache is wiped every time the page refreshes.

Also it's being used in `gutenberg_get_global_stylesheet` and `gutenberg_get_global_stylesheet` which are both using caches as well.

I could be missing some subtleties so please let me know!